test/multiverse/suites/grpc/grpc_client_test.rb

```diff
@@ -36,6 +36,7 @@ def successful_grpc_client_issue_request_with_tracing(metadata = {})
       grpc_client = basic_grpc_client
       trace_with_newrelic_true(grpc_client)
       result = grpc_client.issue_request_with_tracing(
+        nil,
```

Comment:
- update the "grpc" minitest suite tests to accomodate all of the changes and
  additions